backend/de.metas.business/src/main/java/de/metas/costing/CostElement.java

```diff
@@ -55,7 +55,7 @@ private CostElement(
 		this.clientId = clientId;
 	}
 
-	public boolean isMaterialCostingMethod()
+	public boolean isMaterialCostType()
```

Comment:
why renaming it from `isMaterialCostingMethod` to `isMaterialCostType`?

Reply:
Because was misleading. The Element cost type can be material type not the costing method